src/main/java/teammates/common/util/Config.java

```diff
@@ -95,7 +95,11 @@ private String getGcsBucketname() {
     }
 
     private String getAppUrl() {
-        return props.getProperty("app.url");
+        return Url.trimTrailingSlash(props.getProperty("app.url"));
+    }
+    
+    public static AppUrl getAppUrl(String relativeUrl) {
```

Comment:
Done with the leading slash documentation.
Anyway, I think throwing `MalformedURLException` everywhere is not really that advantageous. `Assumption.fail` works in a similar manner (in either case it will be server error), and the URLs are internally generated (no real user involved, except if you count the query values that is) and guarded with all sorts of test cases covering the URL generations of the main application, so that the chances of error is actually pretty slim. Besides, using `Assumption.fail` to wrap Exceptions is quite a common practice, particularly in logic and storage layer. In util (so that it won't be too far off from `Url`), `TimeHelper.convertToDate` is one such case, using `Assumption.fail` instead of throwing `ParseException` everywhere.